README.md

```diff
@@ -201,10 +201,10 @@ to your githooks folder or run `just githooks-install` to copy them all.
 
 Since the altcoin landscape includes projects which [frequently appear and disappear, and are poorly
 designed anyway](https://download.wpsoftware.net/bitcoin/alts.pdf) we do not support any altcoins.
-Supporting Bitcoin properly is already difficult enough and we do not want to increase the
+Supporting Bitcoin properly is already difficult enough, and we do not want to increase the
 maintenance burden and decrease API stability by adding support for other coins.
 
-Our code is public domain so by all means fork it and go wild :)
+Our code is public domain, so by all means, fork it and go wild :)
```

Comment:
The second comma shouldn't be there, and the first doesn't really make it better.